### PR TITLE
TC_04.001.04 | Folder > Rename Folder| Verify to rename the folder from drop-down menu of the folder element in the breadcrumbs

### PR DESCRIPTION
**Pull request by Ridgewood2016**
Implemented Changes:

Converted TC_04.001.04 to POM format.
https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/633

### CODE REVIEW

**Review by antonbloha — COMMENTED** [review]
Add semicolons at the end of the line.

File: cypress/e2e/folderRenameFolderPOM.cy.js [hunk 1]
```diff
@@ -19,11 +19,11 @@ describe('US_04.001 | Folder > Rename Folder', () => {
         dashboardPage.clickNewItemMenuLink();
         newJobPage.typeNewItemName(folderName.name).selectFolder().clickOKButton();
         folderPage.clickSaveButton();
-        header.clickJenkinsLogo();
```
Comment (yanaoreshkevich) [inline cypress/e2e/folderRenameFolderPOM.cy.js:22]: why did you remove this line of code? TC starts with Dashboard page, you can simply add the required line to your code to start your TC and do not change other tests. Please remove all back!

**Review by yanaoreshkevich — CHANGES_REQUESTED** [review]

**Review by Sandarella — DISMISSED** [review]
Сообщение коммита должен содержать название it, в твоем случае: 'TC_04.001.04 |Verify to rename the folder from drop-down menu of the folder element in the breadcrumbs'

1. Восстанови строку 22 как подметила @yanaoreshkevich и закоммить в с правильным содержанием коммита. 
2. Оформи правильно карточку RF и добавь ссылку на нее в описание этого пиар.
3. Добавь предусловия в карточку с тесткейсом.

File: cypress/e2e/folderRenameFolderPOM.cy.js (hunk 1 above)
Comment (Ridgewood2016) [reply cypress/e2e/folderRenameFolderPOM.cy.js:22]: @yanaoreshkevich    I removed a line from a precondition in the file folderRenameFolder.cy.js and added this step to all other necessary tests. However, TC_04.001.04 does not execute with this step: header.clickJenkinsLogo(), as per the requirements of US_04.001 | Folder > Rename Folder.

Acceptance Criteria:
The user should be able to rename the folder using the drop-down menu in the breadcrumbs of the folder element.

Clicking the logo prevents the user from renaming the folder name through the breadcrumbs.
Please let me know if you need further clarification, and we can schedule a call to resolve this issue.

**Comment by Ridgewood2016** [comment]
@Sandarella fixed

**Comment by Ridgewood2016** [comment]
@antonbloha Could you please clarify, what exactly the number line? Thank you

File: cypress/e2e/folderRenameFolderPOM.cy.js (hunk 1 above)
Comment (adiuzhikov) [reply cypress/e2e/folderRenameFolderPOM.cy.js:22]: Hey @yanaoreshkevich, I totally agree with @Ridgewood2016. This line shouldn’t be in beforeEach().

**Review by adiuzhikov — APPROVED** [review]

**Review by yanaoreshkevich — APPROVED** [review]
